eclair-node/src/main/scala/fr/acinq/eclair/Textui.scala

```diff
@@ -27,8 +27,8 @@ class Textui(kit: Kit) extends Logging {
   import com.googlecode.lanterna.gui2._
   import com.googlecode.lanterna.screen.TerminalScreen
   import com.googlecode.lanterna.terminal.DefaultTerminalFactory
-  // Setup terminal and screen layers// Setup terminal and screen layers
 
+  // Setup terminal and screen layers
```

Comment:
should be applied to `wip-android` branch (`wip-android-nosig` is a temp branch)